### PR TITLE
feat: update readme with demo gif and information about project

### LINKED ISSUE #160: Update README with demo gif and information about project
I will provide a script to @cguedes. At a high level I'd like to demonstrate uploading references, creating content in the editor (including exercising all of the fun new features for collapsible blocks etc.), configuring AI interactions (will need to blur the OpenAI key I guess), and sending content to the AI for rewriting and chatting.

### PR DESCRIPTION
**Pull request by cguedes**
This closes #160 

The PR adds:
* Information about the project
* Badges (for CI and code coverage)
* Remove $ in bash scripts
* New section with prerequisites and rewrite of setup section

The new README is available [here](https://github.com/refstudio/refstudio/tree/160-update-readme-with-demo-gif-and-information-about-project#readme).

### CODE REVIEW

**Comment by hammer** [comment]
Good to merge?

**Comment by cguedes** [comment]
I’m trying to get a short GIF now that we have the completion command merged. But we can merge this and I can create another PR only for that.

File: README.md [hunk 1]
```diff
@@ -1,53 +1,119 @@
-# Reference Studio
+<p align="center">
+  <a href="https://github.com/refstudio/refstudio/actions/workflows/on-push.yml">
+    <img src="https://github.com/refstudio/refstudio/actions/workflows/on-push.yml/badge.svg" />
```
Comment (sehyod) [inline README.md:3]: I think there is a missing `</a>` here

Comment (cguedes) [reply README.md:3]: Fixed

**Review by sehyod — APPROVED** [review]

**Pull request merged**